INSTALL.rst

```diff
@@ -23,7 +23,15 @@ Next, install and build assets:
    (inspirehep)$ inspirehep assets build   # NOTE: Might give an error about missing `minjs`, pip install it to fix.
 
 
-Next, create the database and database tables if you haven't already done so:
+Next, create the ElasticSearch Indices and Aliases (this should be done before database init for collection percolators to work):
+
+.. code-block:: console
+
+   (inspirehep)$ inspirehep index init
+
+**NOTE:** you have to install the analysis-icu plugin for this command to work.
+
+Next, create the database and database tables:
```

Comment:
Oh, I see, this commit introduces a different ordering of ES creation/DB creation. Then this doc change should stay in this commit.